docs/react-jss.md

```diff
@@ -36,17 +36,16 @@ yarn add react-jss
 ### Usage
 
 React-JSS wraps your component with a [higher-order component](https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750).
-It injects a `classes` prop, which is a simple map of rule names and generated class names. It can act both as a simple wrapping function and as an [ES7 decorator](https://github.com/wycats/javascript-decorators).
+It injects a `classes` prop, which is a simple map of rule names and generated class names.
 
 Try it out in the [playground](https://codesandbox.io/s/j3l06yyqpw).
 
 #### Basic
 
-```javascript
+```js
```

Comment:
I think we used "javascript" in the most cases, lets stick with something consistent

Reply:
or is it prettier?

Reply:
No, it's not prettier